examples/rust/get_started/examples/04-udp-transport-initiator.rs

```diff
@@ -9,7 +9,10 @@ async fn main(mut ctx: Context) -> Result<()> {
     let _udp = UdpTransport::create(&ctx).await?;
 
     // Send a message to the "echoer" worker, on a different node, over an udp transport.
-    let r = route![(UDP, "localhost:4000"), "echoer"];
+    // Wait to receive a reply and print it.
+    // Use port 4000, unless otherwise specified by command line argument.
+    let port = std::env::args().nth(1).unwrap_or_else(|| "4000".to_string());
+    let r = route![(UDP, format!("localhost:{port}")), "echoer"];
```

Comment:
let's change this to fixed ports as well

Reply:
@mrinalwadhwa done